tty-markdown.gemspec

```diff
@@ -29,7 +29,7 @@ Gem::Specification.new do |spec|
 
   spec.add_dependency "kramdown",   ">= 1.16.2", "< 3.0"
   spec.add_dependency "pastel",     "~> 0.8"
-  spec.add_dependency "rouge",      "~> 3.14"
+  spec.add_dependency "rouge",      ">= 3.14", "< 4.2"
```

Comment:
thought: I went with `"< 4.2"` because I tested (manually and ran the specs) against `rouge 4.1.0` and it worked fine, so it should be OK to at least then.

Reply:
I'd take a leap of faith here and further relax the constraint to `"< 5.0"`. Otherwise, this may cause a bit of churn having to nudge the upper limit up whenever new versions of `4.x` are released.

Reply:
Updated :+1: